### PR TITLE
Update delete-rgs.azcli

### PR DESCRIPTION
**Pull request by jesseloudon**
Added alternative query to show example of specifying a specific tag key/value ( actually used this recently! 😄 )

### CODE REVIEW

File: delete-rgs.azcli [hunk 1]
```diff
@@ -10,7 +10,14 @@
 # This version will ask you to confirm each resource group deletion
 #az group list --tag delete --query [].name -o tsv | xargs -otl az group delete --no-wait  -n
 
+# Alternative query filtering for specific tag key/value | This version will ask you to confirm each resource group deletion
+#az group list --query "[?tags.CostCode=='HashiTalksANZ-2021'].name" -o tsv | xargs -otl az group delete --no-wait -n
```
Comment (jongio) [inline delete-rgs.azcli]: Please use a generic tag like foo=bar

File: delete-rgs.azcli [hunk 2]
```diff
@@ -10,7 +10,14 @@
...
 # This version will NOT ask you to confirm each resource group deletion
 # USE WITH CARE!!!!
 
-az group list --tag delete --query [].name -o tsv | xargs -otl az group delete -y --no-wait  -n
\ No newline at end of file
+az group list --tag delete --query [].name -o tsv | xargs -otl az group delete -y --no-wait  -n
+
+# This version will NOT ask you to confirm each resource group deletion
+# USE WITH CARE!!!!
+az group list --query "[?tags.CostCode=='HashiTalksANZ-2021'].name" -o tsv | xargs -otl az group delete -y --no-wait -n
```
Comment (jongio) [inline delete-rgs.azcli]: Please comment this out and update to generic tag.

**Review by jongio — CHANGES_REQUESTED** [review]
thx for PR!

File: delete-rgs.azcli (hunk 1 above)
Comment (jesseloudon) [reply delete-rgs.azcli]: cheers done

File: delete-rgs.azcli (hunk 2 above)
Comment (jesseloudon) [reply delete-rgs.azcli]: 👍🏻

**Review by jongio — APPROVED** [review]

**Comment by jongio** [comment]
Thanks @jesseloudon

**Pull request merged**